pycon/templates/finaid/receipt_form.html

```diff
@@ -0,0 +1,65 @@
+{% extends "site_base.html" %}
+
+{% load bootstrap_tags %}
+{% load compress %}
+{% load static %}
+{% load i18n %}
+{% load boxes_tags %}
+
+{% block head_title %}Receipt Form{% endblock %}
+
+{% block body_id %}receipt_form{% endblock %}
+
+{% block page_title %}Receipt Application{% endblock %}
+
+{% block body %}
+    {% if form.errors %}
+        <div class="alert">
+            <p class="text-error">
+                There were errors in your form, please correct them and submit again.
+            </p>
+        </div>
+    {% endif %}
+
+    <form method="POST" action="" enctype="multipart/form-data" class="form-horizontal">
+        {% csrf_token %}
+        {% if form.errors and not form.non_field_errors %}
```

Comment:
Good question.
It is also used for editing financial aid applications (templates/finaid/edit.html), so I used it here too; it appears to only display the errors as long as they are errors for a particular field, rather than general form errors. I guess we'd only want to display errors to the user if they are errors with the user's input?